coordinator/src/main/java/org/apache/uniffle/coordinator/web/resource/ServerResource.java

```diff
@@ -171,13 +179,16 @@ public Response<Map<String, Integer>> getNodeStatusTotal() {
     return execute(
         () -> {
           ClusterManager clusterManager = getClusterManager();
-          Map<String, Integer> stringIntegerHash =
-              Stream.concat(
-                      clusterManager.getServerList(Collections.emptySet()).stream(),
-                      clusterManager.getLostServerList().stream())
-                  .collect(
-                      Collectors.groupingBy(
-                          n -> n.getStatus().name(), Collectors.reducing(0, n -> 1, Integer::sum)));
+          Map<String, Integer> stringIntegerHash = Maps.newHashMap();
+          stringIntegerHash.put("lostNode", clusterManager.getLostServerList().size());
+          stringIntegerHash.put("unhealthyNode", clusterManager.getUnhealthyServerList().size());
+          stringIntegerHash.put(
+              "activeNode", clusterManager.getServerList(Collections.emptySet()).size());
```

Comment:
`getServerList` is not only return active nodes.

Reply:
Why do you use the earlier logic?
https://github.com/apache/incubator-uniffle/pull/1053#discussion_r1299893603

Reply:
> Why do you use the earlier logic? [#1053 (comment)](https://github.com/apache/incubator-uniffle/pull/1053#discussion_r1299893603)

@yl09099 Are you miss this thread?

Reply:
> > Why do you use the earlier logic? [#1053 (comment)](https://github.com/apache/incubator-uniffle/pull/1053#discussion_r1299893603)
> 
> @yl09099 Are you miss this thread?

Have been modified.

Reply:
I mean why do we need to modify this part of the logic?  As we discuss in https://github.com/apache/incubator-uniffle/pull/1053#discussion_r1298078135, If we add status in the future, we need to modify this method.

Reply:
> I mean why do we need to modify this part of the logic? As we discuss in [#1053 (comment)](https://github.com/apache/incubator-uniffle/pull/1053#discussion_r1298078135), If we add status in the future, we need to modify this method.

![image](https://github.com/apache/incubator-uniffle/assets/33595968/de79a375-d2b3-4d97-802c-5da71735e1c8)
Provide a display of the number of servernode states here.

Reply:
> I mean why do we need to modify this part of the logic? As we discuss in [#1053 (comment)](https://github.com/apache/incubator-uniffle/pull/1053#discussion_r1298078135), If we add status in the future, we need to modify this method.

The layout of the status page also has to be changed, it doesn't have to be that way.